recipes/atlas-schema/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "atlas-schema" %}
+{% set version = "0.1.0" %}
```

Comment:
There is a 0.2.0 on PyPI. Do you need the 0.1? If not, please add the latest one.

Reply:
It would be nice to have parity between PyPI and conda-forge, so having v0.1 could be useful for us.